avicii.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "PSavarMattas",
+			"oem": "OnePlus",
+			"device": "OnePlus Nord (Avicii)",
+			"filename": "crDroidAndroid-13.0-20230720-avicii-v9.7.zip",
+			"download": "https://sourceforge.net/projects/crdroidandroid-avicii/files/crDroidAndroid-13.0-20230720-avicii-v9.7.zip/download",
+			"timestamp": 1689863641,
+			"md5": "61d3ef948b8652d702de940b5a2590fa",
+			"sha256": "96a73afe5003efdec76513cb73dfbcd5cc55daac3be429ddcd66ad72c66c1df8",
+			"size": 1306214305,
+			"version": "9.7",
+			"buildtype": "Active Development Releases",
```

Comment:
use any of below
Testing/Alpha/Beta/Weekly/Monthly
as per https://github.com/crdroidandroid/android_vendor_crDroidOTA/blob/13.0/README.md#11-json-structure